portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,211 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+const RESUME = {
```

Comment:
nit: per the [style-guide](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes), can you change all the quotes to be single quotes?

Reply:
Done in latest commit